sdk/storage/CMakeLists.txt

```diff
@@ -10,12 +10,6 @@ set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 
 option(BUILD_STORAGE_SAMPLES "Build storage sample codes" ON)
 
-
-if(MSVC)
```

Comment:
what's the motivation of removing this?

Reply:
I listed in PR description - it is better not to rely on the fact that our project is always built via cmake, and when we need to enable vcpkg build (#1013), we need individually buildable directories, which means that we can't rely on the fact that, for example, `storage/azure-storage-common/CMakeLists.txt` was loaded from the `storage/CMakeLists.txt` and therefore all the definitions from the broader directory file are present when building azure-storage-common.

Reply:
I think you're right